library/std/src/io/error.rs

```diff
@@ -361,6 +361,22 @@ impl ErrorKind {
     }
 }
 
+#[stable(feature = "io_errorkind_display", since = "1.60.0")]
+impl fmt::Display for ErrorKind {
+    /// Shows a human-readable description of the `ErrorKind`.
+    ///
+    /// This is similar to `impl Display for Error`, but doesn't require the error to be allocated on the heap.
```

Comment:
I realized after writing this that the comment is wrong and this impl is less useful than I thought, because `impl From<ErrorKind> for Error` doesn't allocate. But this still seems like a useful change even so to avoid noise.